Bor_Trie/Bor_Trie/Program.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bor_Trie
+{
+    public class Trie
+    {
+        public class Node
+        {
+            public bool Terminal { get; set; }
+            public Dictionary<char, Node> Children { get; set; }
+            public int Number { get; set; }
+            public Node()
+            {
+                Terminal = false;
+                Children = new Dictionary<char, Node>();
+                Number = 0;
+            }
+        }
+
+        public Node root;
+        public int Size { get; set; }
+        public Trie()
+        {
+            Size = 0;
+            root = new Node();
+        }
+        public bool Add(string s)
+        {
+            Node Chek_Node = root;

```

Comment:
```suggestion
            Node CheckNode = root;

```
опять же, почитай styleguide: https://google.github.io/styleguide/csharp-style.html